webapp/src/selectors/index.tsx

```diff
@@ -6,6 +6,14 @@ export const getRhsState = (state: ReduxState): {isSidebarOpen: boolean} => {
     return state.views.rhs;
 };
 
+export const getGlobalModalState = (state: ReduxState): GlobalModalState => {
+    return state['plugins-mattermost-plugin-azure-devops'].globalModalSlice;
```

Comment:
Can't you use a constant here for the plugin id?

Reply:
this is already a kind of constant as we have defined this as a type in `ReduxState` so it's a string literal and TS will not allow any value other than this